src/registry.c

```diff
@@ -1146,9 +1169,11 @@ validate(struct rxkb_context *ctx, xmlDoc *doc)
     xmlValidCtxt *dtdvalid = NULL;
     xmlDtd *dtd = NULL;
     xmlParserInputBufferPtr buf = NULL;
-    /* This is a modified version of the xkeyboard-config xkb.dtd. That one
-     * requires modelList, layoutList and optionList, we
-     * allow for any of those to be missing.
+    /* This is a modified version of the xkeyboard-config xkb.dtd:
+     * • xkeyboard-config requires modelList, layoutList and optionList,
+     *   but we allow for any of those to be missing.
+     * • xkeyboard-config sets default value of “popularity” to “standard”,
+     *   but we set this value depending if we are loading an “extras” rule file.
```

Comment:
```suggestion
     *   but we set this value depending if we are currently parsing an “extras” rule file.
```
slightly less ambiguous

Reply:
Fixed